### PR TITLE
ci: update build image

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Loong Dai <loong.dai@intel.com>

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA] #21467
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -284,7 +284,7 @@ build:remote-clang-cl --config=rbe-toolchain-clang-cl
 
 # Docker sandbox
 # NOTE: Update this from https://github.com/envoyproxy/envoy-build-tools/blob/main/toolchains/rbe_toolchains_config.bzl#L8
```
Comment (daixiang0) [inline .bazelrc:286]: I check this and find it a long time no update.

I am not sure what kind of process update should I use now.

/cc @phlax @lizan

**Comment by phlax** [comment]
@daixiang0 i think this is a dupe of #21388 which has a blocker that needs to be figured out

**Comment by daixiang0** [comment]
Oh, I see it.

I remove the confused comment here.

**Comment by daixiang0** [comment]
https://dev.azure.com/cncf/envoy/_build/results?buildId=109380&view=logs&j=8c169225-0ae8-53bd-947f-07cb81846cb5&t=6b0ace90-28b2-51d9-2951-b1fd35e67dec&l=1381 emscripten build failed, I met it for some times, seems it flaky.

File: .bazelrc [hunk 2]
```diff
@@ -283,7 +283,6 @@ build:remote-clang-cl --config=clang-cl
 build:remote-clang-cl --config=rbe-toolchain-clang-cl
 
 # Docker sandbox
-# NOTE: Update this from https://github.com/envoyproxy/envoy-build-tools/blob/main/toolchains/rbe_toolchains_config.bzl#L8
```
Comment (phlax) [inline .bazelrc:286]: im wondering why remove this - it seems like that line is the authoritative place to get the image hash from

**Comment by phlax** [comment]
> emscripten build failed, I met it for some times, seems it flaky.

see conversations/threads on slack#envoy-ci

**Comment by htuch** [comment]
Please merge main to see if the CI fixes help.

File: .bazelrc (hunk 2 above)
Comment (daixiang0) [reply .bazelrc:286]: No update for several months, not sure how image upgrade works.

**Comment by daixiang0** [comment]
> Please merge main to see if the CI fixes help.

I will try.

**Comment by phlax** [comment]
@daixiang0 im closing this - the info here may not be best placed or comprehensive, but it _is_ correct and there is not a good reason to remove it

**Pull request closed without merging**